test/TestTaskIncludes.py

```diff
@@ -56,3 +56,10 @@ def test_include_tasks_in_role(self):
         runner = Runner(self.rules, filename, [], [], [])
         runner.run()
         self.assertEqual(len(runner.playbooks), 4)
+
+    @unittest.skipIf(parse_version(ansible.__version__) < parse_version('2.7'), "not supported with ansible < 2.7")
+    def test_import_tasks_2_7_style(self):
+        filename = 'test/taskincludes_2_7_style.yml'
+        runner = Runner(self.rules, filename, [], [], [])
+        runner.run()
+        self.assertEqual(len(runner.playbooks), 2)
```

Comment:
Now that we use pytest runner, it's recommended to use `assert`
```suggestion
        assert len(runner.playbooks) == 2
```

Reply:
I do remember that assert was not encouraged for several reasons but I forgot why. I think one may be the way pytest displays a potential assert failure?

On the other hand I do like the clearness of a simple assert statement.

Reply:
`assert` is not encouraged in runtime because it's stripped off in the optimized mode (`python -O`). Under the hood, it's implemented as
```python
if __debug__:
    raise AssertError if needed
```
And `__debug__` is `False` in the optimized mode.

This was valid for nose/unittest runners but `pytest` specifically relies on `assert`s and it also rewrites the bytecode for them.
Displaying extra assertion comparison is another feature with arises from this.